src/common/components/routes/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Route } from 'react-router'
+
+function RouteWithSubRoutes(route) {
+  return (
+    <Route
+      path={route.path}
+      render={props => <route.component {...props} routes={route.routes} />}
+    />
+  )
+}
+
```

Comment:
`const RouteWithSubRoutes = ({ component, path, routes }) => 
     <Route path= { path } render={props => <component { ...props } routes={ routes }}/> />`

Reply:
It doesn't work.  Function doesn't use prop 'component'

Reply:
could you paste the exact error please

Reply:
![image](https://user-images.githubusercontent.com/34004410/70135330-8aec4e00-1692-11ea-8e8c-76440ce28a48.png)

Reply:
unfortunately this is just a screenshot and not descriptive at all, maybe you could hover over args and get some error info ......... just wondering why it doesn't work, btw you can omit this